src/lib/charts/PieChart/PieChart.tsx

```diff
@@ -1,26 +1,21 @@
 import React, { FC } from 'react';
 import { merge } from 'lodash';
 import classNames from 'classnames';
-import { ChartDataFunction } from 'react-chartjs-2';
 import { ChartBase, ChartBaseProps } from '../ChartBase';
-import { ChartJSData, ChartJSOptions } from '../chart.helpers';
+import { getComputedPieChartJSData, getPieChartPercentage, PieChartData } from '../chart.helpers';
+import { ChartOptions } from 'chart.js';
 
 export interface PieChartProps extends Partial<ChartBaseProps> {
-  data?: {
-    label: string;
-    value: number;
-    color: string;
-  }[];
+  data?: PieChartData;
 }
 
-export const PieChart: FC<PieChartProps> = ({ chartJSData, data, options, className, ...props }) => {
-  const defaultOptions: ChartJSOptions = {
+export const PieChart: FC<PieChartProps> = ({ className, options, chartJSData, data, ...props }) => {
+  const computedChartJSData = getComputedPieChartJSData(chartJSData, data);
+
+  const defaultOptions: ChartOptions = {
     cutoutPercentage: 55,
     legend: {
-      onClick: () => {
-        // The default behavior is to toggle data points when the legend item is clicked, but since this is a pie chart
-        // we don't really need that behavior.
-      }
+      display: false
```

Comment:
I went ahead and made this the default in the `ChartBase`, since I went ahead and created a good default legend.